src/Form/FormAnalyzer.js

```diff
@@ -130,7 +130,7 @@ class FormAnalyzer {
     getText (el) {
         // for buttons, we don't care about descendants, just get the whole text as is
         // this is important in order to give proper attribution of the text to the button
-        if (this.elementIs(el, 'BUTTON')) return el.innerText
+        if (this.elementIs(el, 'BUTTON')) return el.textContent
```

Comment:
Using `innerText` here meant we might have missed the contents of a hidden button. It fixes: https://app.asana.com/0/1199178362774117/1201428705646860/f. Also, again, `textContent` is more performant so it's preferable unless there is a reason.